src/coreComponents/constitutive/fluid/multifluid/CO2Brine/functions/PVTFunctionHelpers.hpp

```diff
@@ -19,6 +19,8 @@
 #include "common/DataTypes.hpp"
 #include "common/Units.hpp"
 
+#include "codingUtilities/StringUtilities.hpp"
```

Comment:
Probably should be in the cpp if the only change in the file

Reply:
This was a little strange to me. `StringUtilities` is used in the `.hpp` file itself. It means that any file that uses `PVTFunctionHelpers.hpp` has to also include `StringUtilities.hpp` otherwise the compilation fails.